chapters/5-processing.tex

```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
 so other researchers may use them for their analysis as well.
 
 \section{Looking ahead}
-Once you have corrected, cleaned, 
-and de-identified one or several data tables,
-and compiled the documentation to support it,
+This chapter introduced a process of exploring, understanding, and correcting
```

Comment:
```suggestion
This chapter introduced a process of exploring, understanding, and processing
```

Reply:
That is redundant with "this process ... is called processing" on Line 833. Which is also poorly written... Far too much "processing" going on here. Do we have other terms available for this?

Reply:
```suggestion
This chapter introduced a workflow of formatting, cleaning, and correcting
```

Do you not like "correcting" specifically?

Reply:
workflow is a good replacement for processing here. correction raises red flags for me, we don't mean it this way, but too often it's 'correcting' the data to make it look more like what you expected ... that's why i wanted to replace.

Reply:
we call it correction throughout the chapter though, should we review that?

Reply:
in context it's ok . i guess fine here too, just makes me nervous! perhaps something like correcting field errors to help limit the scope?

Reply:
how about QA as now suggested.